src/docs/sdk/performance/span-operations.mdx

```diff
@@ -0,0 +1,130 @@
+| angular  |              | Spans related to [Angular](https://angular.io/)  |
+| ember    |              | Spans related to [EmberJS](https://emberjs.com/) |
+
+## Web Server
+
+| Category  | Usage           | Description |
+| --------- | --------------- | ----------- |
+| http      |                 |             |
+|           | http.client     |             |
+|           | http.server     |             |
+| rpc       |                 |             |
+| grpc      |                 |             |
+| template  |                 |             |
+|           | template.init   |             |
+|           | template.parse  |             |
+|           | template.render |             |
+| render    |                 |             |
+|           | view.render     |             |
+| serialize |                 |             |
+| console   |                 |             |
+
+## Web Frameworks
+
+| Category | Usage             | Description |
+| -------- | ----------------- | ----------- |
+| django   |                   |             |
+|          | django.middleware |             |
+|          | django.view       |             |
+| express  |                   |             |
+| rails    |                   |             |
+| rack     |                   |             |
+
+## Database
+
+| Category | Usage            | Description                |
+| -------- | ---------------- | -------------------------- |
+| db       |                  | An operation on a database |
+|          | db.connection    |                            |
+|          | db.transaction   |                            |
+|          | db.sql.query     |                            |
+|          | db.query         |                            |
+|          | db.query.compile |                            |
+
+## Serverless (FAAS)
+
+| Category | Usage            | Description |
+| -------- | ---------------- | ----------- |
+| faas     |                  |             |
+|          | faas.aws         |             |
+|          | faas.aws.lambda  |             |
+|          | faas.aws.request |             |
+|          | faas.gcp         |             |
+
+## Mobile
+
+| Category   | Usage          | Description                         |
+| ---------- | -------------- | ----------------------------------- |
+| app        |                | Data about the mobile app           |
+|            | app.start      |                                     |
+|            | app.start.warm |                                     |
+|            | app.start.cold |                                     |
+| ui         |                | An operation on a mobile/desktop UI |
+|            | ui.load        |                                     |
+|            | ui.action      |                                     |
+| navigation |                | Navigating to another screen        |
+| file       |                | Operations on the file system       |
```

Comment:
In a call, we discussed what to put in the description of the file span. We thought that filename + bytes count `file.txt (203 KB)` should go in the description and the full file path should go into data of the span as `data["filepath"] = "/absolute/example/file.txt"`. Should the bytes count be KB, MB, GB, ... ?

Can we document this here?

Reply:
> Can we document this here?

We should - but I'm not going to include it in this PR.

Instead we should be trying to develop our own set of semantic conventions (especially for desktop/mobile/browser - areas where open telemetry doesn't have anything). I plan to take this on after this PR gets merged in.

Reply:
Okay, I created an issue so this doesn't get lost https://github.com/getsentry/develop/issues/463